src/current/cockroachcloud/production-checklist.md

```diff
@@ -22,27 +76,53 @@ For guidance on sizing, validating, and using connection pools with CockroachDB,
+
+For CockroachDB {{ site.data.products.dedicated }} clusters, Cockroach Labs runs [full cluster backups](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/take-full-and-incremental-backups#full-backups) daily and [incremental cluster backups](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/take-full-and-incremental-backups#incremental-backups) hourly. The full backups are retained for 30 days, while incremental backups are retained for 7 days. Once a cluster is deleted, Cockroach Labs retains the full backups for 30 days and incremental backups for 7 days. Backups are stored in the same region that a single-region cluster is running in or the primary region of a multi-region cluster.
+
+You can restore cluster data to the current cluster or another cluster in the same organization. You can also restore a database or table from the Backups tab.
+
+{{site.data.alerts.callout_danger}}
+Restoring to a cluster will completely erase all data in the destination cluster. All cluster data will be replaced with the data from the backup. The destination cluster will be unavailable while this operation is in progress. This operation cannot be canceled, paused, or reversed.
+{{site.data.alerts.end}} 
+
+You can [manage your own backups]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}), including incremental, database, and table level backups. To perform manual backups, you must configure either a userfile location or a cloud storage location.
+
+## Patches and upgrades
+
+CockroachDB {{ site.data.products.cloud }} supports the latest major version of CockroachDB and the version immediately preceding it. Support for these versions includes patch version updates and security patches.
+
+### Major version upgrades
+
+Major version upgrades are automatic for CockroachDB {{ site.data.products.serverless }} clusters and opt-in for CockroachDB {{ site.data.products.dedicated }} clusters. [Cluster Operators]({% link cockroachcloud/authorization.md %}#cluster-operator) must initiate major version upgrades for CockroachDB {{ site.data.products.dedicated }} clusters. When a major version upgrade is initiated for a cluster, it will upgrade to the latest patch version as well. 
+
+Since upgrading a cluster can have a significant impact on your workload, make sure you review the release notes for the latest version for backward compatibility, cluster setting changes, deprecations, and known limitations. Cockroach Labs recommends initiating the upgrade during off-peak periods and monitor cluster and application health post-upgrade. If you notice functional or performance regression, you will have 72 hours to roll back the changes before the upgrade is automatically finalized. Note that some new features might be unavailable until the upgrade is finalized. For more information, see [Major version upgrades]({% link cockroachcloud/upgrade-policy.md %}#major-version-upgrades).
```

Comment:
```suggestion
Since upgrading a cluster can have a significant impact on your workload, make sure you review the release notes for the latest version for backward compatibility, cluster setting changes, deprecations, and known limitations. Cockroach Labs recommends initiating the upgrade during off-peak periods. After the upgrade, carefully monitor cluster and application health. If you notice functional or performance regression, you can roll back the changes for up to 72 hours before the upgrade is automatically finalized. After an upgrade, some features might be unavailable until the upgrade is finalized. For more information, refer [Major version upgrades]({% link cockroachcloud/upgrade-policy.md %}#major-version-upgrades).
```